recipe/meta.yaml

```
@@ -10,18 +10,23 @@ source:
   sha256: 97bd9f8dcceeb835e0fdc0e2a89eeba0d566e17f9abbed6ee0003c4b3b356cda
 
 build:
+  number: 0
   entry_points:
     - whey = whey.__main__:main
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
-  number: 1
+    - whey_sdist = whey.builder:SDistBuilder
+    - whey_wheel = whey.builder:WheelBuilder
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True  # [py<37]
```

```suggestion
  skip: True  # [py<36]
```
https://github.com/repo-helper/whey/blob/0bebadcdaafb17ee3b562a6e8a0e08618014bed0/pyproject.toml#L159

---

done